{"name": "Проверка ручки /api/v1/courier. 1.нельзя создать двух одинаковых курьеров;2.если создать пользователя с логином, который уже есть, то возвращается ошибка.3.запрос возвращает код ответа 409 и содержит текст \"Этот логин уже используется\".", "status": "passed", "start": 1723407918047, "stop": 1723407918416, "uuid": "65988585-29d0-41c4-a6ae-0f2e65e74fad", "historyId": "3a6f8cb8569b4bba538d1c820845b87c", "testCaseId": "3a6f8cb8569b4bba538d1c820845b87c", "fullName": "tests.tests_courier.test_courier_creation.TestCourierCreation#test_create_courier_with_existing_login_shows_error_409", "labels": [{"name": "parentSuite", "value": "tests.tests_courier"}, {"name": "suite", "value": "test_courier_creation"}, {"name": "subSuite", "value": "TestCourierCreation"}, {"name": "host", "value": "DESKTOP-7ROOEBR"}, {"name": "thread", "value": "11600-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.tests_courier.test_courier_creation"}]}